frontend/app/playwright.config.ts

```diff
@@ -16,23 +16,21 @@ export default defineConfig({
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
-  retries: process.env.CI ? 1 : 0,
+  retries: 0,
   /* 30s timeout for both assertions and tests (2min in CI) */
   timeout: process.env.CI
     ? process.env.INFRAHUB_MISC_RESPONSE_DELAY
-      ? 4 * 60 * 1000
-      : 2 * 60 * 1000
+      ? 5 * 60 * 1000
+      : 3 * 60 * 1000
     : 30 * 1000,
   expect: {
     timeout: process.env.CI
       ? process.env.INFRAHUB_MISC_RESPONSE_DELAY
-        ? 4 * 60 * 1000
-        : 2 * 60 * 1000
+        ? 6 * 60 * 1000
+        : 3 * 60 * 1000
       : 30 * 1000,
     toHaveScreenshot: { maxDiffPixels: 5000 },
   },
-  /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 4 : undefined,
```

Comment:
We can keep 4 workers for the moment, I think it's fine.

Reply:
Yep I was just curious to see what happens when nothing is specified. It was not plan to be merged

Reply:
FYI, it uses 16 workers in this case :p